src/Store/StoreThread.re

```diff
@@ -115,8 +115,16 @@ let start =
     );
   let themeUpdater = ThemeStoreConnector.start(themeInfo);
 
+  // TODO: This is only necessary because the config file can't be passed to the initialization in order for the state to be properly initalized.
+  let config =
+    Feature_Configuration.update(
+      ~configFile=configurationFilePath,
+      state.config,
+      ConfigurationFileChanged,
+    );
+
```

Comment:
This is pretty bad, but is currently needed because the state is currently initialized in several different places and can't be passed any initial arguments because of that. I've bumped into this limitation several times now, so I might try to tackle that before merging this, so we can remove this hack.

Reply:
Thanks for fixing this in #1490

Reply:
Fixed in https://github.com/onivim/oni2/pull/1489/commits/6d6d7f83fb3d3c562dde77d155443b1522a2ea65